cli/Cargo.toml

```diff
@@ -46,104 +46,105 @@ parking_lot = "0.11.1"
 pallet-credit = {version = "3.0.0", default-features = false, path = "../pallets/credit"}
 pallet-micropayment = {version = "3.0.0", default-features = false, path = "../pallets/micropayment"}
 
 # frontier frame dependencies
-pallet-evm = { git = "https://github.com/deeper-chain/frontier.git", rev = "8b1134332d8777f3bee68f3f95ec82e40e736e78" }
-pallet-ethereum = { git = "https://github.com/deeper-chain/frontier.git", rev = "8b1134332d8777f3bee68f3f95ec82e40e736e78" }
-pallet-dynamic-fee = { git = "https://github.com/deeper-chain/frontier.git", rev = "8b1134332d8777f3bee68f3f95ec82e40e736e78" }
-pallet-base-fee = { git = "https://github.com/deeper-chain/frontier.git", rev = "8b1134332d8777f3bee68f3f95ec82e40e736e78" }
+pallet-evm = { git = "https://github.com/deeper-chain/frontier.git", branch = "feature/polkadot0.9.16_dev" }
+pallet-ethereum = { git = "https://github.com/deeper-chain/frontier.git", branch = "feature/polkadot0.9.16_dev" }
+pallet-dynamic-fee = { git = "https://github.com/deeper-chain/frontier.git", branch = "feature/polkadot0.9.16_dev" }
+pallet-base-fee = { git = "https://github.com/deeper-chain/frontier.git", branch = "feature/polkadot0.9.16_dev" }
 
 # node-specific dependencies
 node-runtime = { version = "2.0.0", path = "../runtime" }
 node-rpc = { version = "2.0.0", path = "../rpc" }
 node-primitives = { version = "2.0.0", path = "../primitives" }
 node-executor = { version = "2.0.0", path = "../executor" }
 
-frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12" }
+frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16" }
 
 # CLI-specific dependencies
-sc-cli = { optional = true, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12" }
-frame-benchmarking-cli = { optional = true, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12" }
+sc-cli = { optional = true, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16" }
+frame-benchmarking-cli = { optional = true, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16" }
 node-inspect = { version = "0.8.0", optional = true, path = "../inspect" }
 
 [target.'cfg(target_arch="x86_64")'.dependencies]
 node-executor = { version = "2.0.0", path = "../executor", features = [ "wasmtime" ] }
-sc-cli = { optional = true, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12", features = [ "wasmtime" ] }
-sc-service = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12", default-features = false, features = [ "wasmtime" ] }
-sp-trie = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.12", default-features = false, features = ["memory-tracker"] }
+sc-cli = { optional = true, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", features = [ "wasmtime" ] }
```

Comment:
sc-cli, sc-service, sc-executor either all have a `features = [ "wasmtime" ]` or none have this feature.

Reply:
yes, all features = [ "wasmtime" ] is seted in cli/Cargo.toml